web_set_single_page_hidden/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This modules modifies `FieldOne2Many` and `FieldMany2Many` widgets:
```

Comment:
I'd add what the final goal is in layman's terms: to always visualise the line total count